lib/godaddy-style.js

```diff
@@ -32,6 +32,12 @@ module.exports = function (gulp, options) {
     });
     return gulp.src(options.files)
       .pipe(eslint(eslintOpts))
+      .pipe(through2.obj(function (file, enc, callback) {
+        if (options.eslint.fix && file.eslint.fixed) {
+          return fs.writeFile(file.path, file.eslint.output, function (err) { callback(err, file); });
+        }
+        callback(null, file);
+      }))
```

Comment:
Can't you use `gulp.dest`?
Like: https://github.com/adametry/gulp-eslint/blob/master/example/fix.js#L14-L23